### DISCUSSION TITLE
[models] Billboard has different size based on rotation

### DISCUSSION THREAD

**Discussion by nobytesgiven** [raysan5/raylib#1798] [Q&A]
### Issue description
Billboards change in size if I rotate them in my image editor. I do not know if this is intentional but it would be nice to have consistent sizing for same sized images.

### Issue Screenshot
![image](https://user-images.githubusercontent.com/80068279/117584602-8a24da80-b116-11eb-816c-9b1932ee14b3.png)
![image](https://user-images.githubusercontent.com/80068279/117584599-85f8bd00-b116-11eb-8cc4-02332bfdb30d.png)

### Code Example
Just the basic billboard example, didn't edit the code.

**Comment by raysan5** [comment]
Excuse me but I don't understand the issue, I just tested `models_billboard` example and it seems to work ok for me.

**Comment by nobytesgiven** [comment]
> 
> 
> Excuse me but I don't understand the issue, I just tested `models_billboard` example and it seems to work ok for me.

I just find it odd how the scaling is only based on the width of the image. I would expect the same image but rotated in my image editor would have consistent sizing when drawn.

**Comment by kayomn** [comment]
If I understand the issue correct, all you have done with the images between those two screenshots is rotated it 90 degrees, correct?

**Comment by nobytesgiven** [comment]
> If I understand the issue correct, all you have done with the images between those two screenshots is rotated it 90 degrees, correct?

yes

**Comment by raysan5** [comment]
@nobytesgiven Did you try with latest raylib from GitHub? `DrawBillboardPro()` was added recently.

**Comment by nobytesgiven** [comment]
Yes, this is **not** a bug, I'm just pointing out that the image scaling is only based on the image width and size (You can see the width on those images is the same). If this seems normal to everyone just close this PR, this is **not priority**. I opened it more like a discussion.
![image](https://user-images.githubusercontent.com/80068279/119042778-f75f2800-b9c0-11eb-9174-170881b7f5ea.png)
![image](https://user-images.githubusercontent.com/80068279/119042819-0514ad80-b9c1-11eb-82a6-348881237419.png)
![billboard](https://user-images.githubusercontent.com/80068279/119042865-165dba00-b9c1-11eb-8052-2559dac2fb2a.png)
![billboard](https://user-images.githubusercontent.com/80068279/119042851-11006f80-b9c1-11eb-8983-44b8d799f6d0.png)

**Comment by nobytesgiven** [comment]
I guess this can be converted to a discussion/deleted, since noone seems to be bothered from it but me.

**Reply by raysan5** [reply]
@nobytesgiven Just converted this issue into a Discussion.

**Comment by kayomn** [comment]
`DrawBillboardPro` accepts a `Rectangle` parameter that defines the bounds of the rectangle, as billboard sprites are drawn in a worldspace scale, not a screen pixel-accurate presentation.

Are you changing this rectangle to reflect the shape you wish to render between the two textures being drawn?
